README.md

```diff
@@ -9,8 +9,15 @@ pip install invenio-analytics-importer
 
 ## Usage
 
+**Retrieve analytics**
+
+```bash
+pipenv run invenio analytics_importer retrieve --from <YYYY-MM-DD> --to <YYYY-MM-DD> --output-dir <path>/<to>/<data>/
```

Comment:
Where are `--from` and `--to` defined/used?

Reply:
In the cli.py script that will come in the next PR or 2. They are command line options.